ci/launch_cluster.sh

```diff
@@ -38,25 +38,34 @@ envsubst <<EOF > "$CONFIG_PATH"
 ---
 launch_config_version: 1
 deployment_name: $DEPLOYMENT_NAME
-installer_url: $INSTALLER
-provider: onprem
-platform: aws
 aws_region: us-west-2
-os_name: cent-os-7-dcos-prereqs
```

Comment:
Was this removed?

Reply:
It's not removed, it's moved down.

I had a weird conflict in this file so I sorted the lines alphabetically to make it easy to compare.